pkg/regressionallowances/intentional_4.15_regressions.go

```diff
@@ -22,4 +24,23 @@ func init() {
 				ReasonToAllowInsteadOfFix: "",
 			})
 	*/
+
+	mustAddIntentionalRegression(
+		release415,
+		IntentionalRegression{
+			JiraComponent: "Networking / openshift-sdn",                       // Jira Component,  not team name
+			TestID:        "cluster install:0cb1bb27e418491b1ffdacab58c5c8c0", // ask TRT for the ID for your TestName
+			TestName:      "install should succeed: overall",                  // this helps approvers recognize at a glance
+			Variant: api.ComponentReportColumnIdentification{ // this indicates the selectivity of the choice
+				Network:  "sdn",
+				Upgrade:  "upgrade-micro",
+				Arch:     "amd64",
+				Platform: "metal-ipi",
+			},
+			PreviousPassPercentage:    100,
+			PreviousSampleSize:        61, // number of runs used for the percentage
+			RegressedPassPercentage:   50,
+			RegressedSampleSize:       12, // number of runs used for the percentage
+			ReasonToAllowInsteadOfFix: "I can't explain this, but Ben Bennett maybe?",
```

Comment:
Is this property expected to link work to fix the regression, when such work is planned?  I could see:

* We regress, and we want to block 4.y patch releases until we fix it.  There's a tracking OCPBUGS-# pointed at 4.y.z.
* We regress, and decide that's just how the world is now.  We land an intentional regression exception and do not remove it until we stop running Sippy on that old 4.y, long after its end-of-life.
* We regress, and decide that we want to fix it, but that it's not bad enough to block further 4.y patch releases.  There's a tracking OCPBUGS-# pointed at 4.y.z and an intentional regression exception that gets removed once we're done touching builds that lack the fix.

It's that last case where I'd expect an OCPBUGS-# link somewhere in the regression declaration, and I'm wondering if that is structured somewhere, or unstructured in this property, or unstructured in the commit message, or...?